roles/backend_setup/tasks/main.yml

```diff
@@ -91,3 +91,10 @@
   when: gluster_infra_thick_lvs is defined and gluster_infra_thick_lvs | selectattr("atboot") | list | length > 0
   tags:
     - lvmkernelparams
+
+# set fstrim service 
+- name: Configure lvm kernel parameters
+  import_tasks: fstrim_service.yml
+  when: fstrim_service is defined
```

Comment:
We follow the convention of appending gluster_infra_ to the variables. So that we don't clash with other roles if we happen to use within the playbook.

And variables names like these are used for local variables within the tasks file.